.github/workflows/release_portable_linux_packages.yml

```diff
@@ -134,15 +134,14 @@ jobs:
           echo "Building ${{ env.DIST_ARCHIVE }}"
           tar cfz "${{ env.DIST_ARCHIVE }}" .
 
-      - name: Build Python Packages
+      - name: Build PyTorch Production Wheels
         run: |
-          ./build_tools/linux_portable_build.py \
-            --image=${{ env.BUILD_IMAGE }} \
-            --output-dir=${{ env.OUTPUT_DIR }}/packages \
-            --build-python-only \
-            --artifact-dir=${{ env.OUTPUT_DIR }}/build/artifacts \
-            -- \
-            "--version=${{ needs.setup_metadata.outputs.version }}"
+          ./external-builds/pytorch/build_prod_wheels.py \
+            --install-rocm \
+            --clean \
+            --output-dir ${{ env.OUTPUT_DIR }}/packages/dist \
+            --pytorch-rocm-arch ${{ matrix.target_bundle.amdgpu_family }} \
```

Comment:
Pytorch needs individual gfx arches, not our combined package families. 

I think I need to extend the wheels so they can report which gfx arches they were built for. Then use that here. But probably better to just build for one arch to start.

Reply:
Sounds good, I used `gfx942`